### ISSUE TITLE
Training on custom dataset

### ISSUE THREAD

**Issue by Aayushktyagi** [wasidennis/AdaptSegNet#78]
Hi ,
Thanks @hfslyc  for sharing code.
I am trying to run the network on the custom dataset. My dataset contains binary mask.
I created custom data loader for both source and target.
I am facing issue with loading model. 
1) I used deep labs with single class I am getting the following error while I am using deep labs with num_classes = 1

![Screenshot from 2020-01-27 15-56-10](https://user-images.githubusercontent.com/32107805/73167360-bf720880-411d-11ea-98c3-f6ab211aaf53.png)

when model loading structure was
![Screenshot from 2020-01-27 17-39-07](https://user-images.githubusercontent.com/32107805/73173684-f6e7b180-412b-11ea-8e78-73146a82bccc.png)

2) Can you please guide me which changes need to be done when training model from scratch.
![Screenshot from 2020-01-27 16-01-08](https://user-images.githubusercontent.com/32107805/73167672-445d2200-411e-11ea-8453-3ab968a1b530.png)

Thanks in advance 
Aayush

_Originally posted by @Aayushktyagi in https://github.com/wasidennis/AdaptSegNet/issues/1#issuecomment-578686821_

**Comment by hfslyc** [comment]
Hi,

for 1, I think you should set the NUM_CLASSES to 2 for a binary mask. Also, please make sure that the gt labels are 0 and 1 instead of the other number.

for 2, if you really want to train from scratch (not using imagenet pretrained model), you could simply comment out all the model.load_state_dict(.)
